crypto/mem_sec.c

```diff
@@ -465,24 +480,43 @@ static int sh_init(size_t size, size_t minsize)
             close(fd);
         }
     }
-#endif
+# endif
     if (sh.map_result == MAP_FAILED)
         goto err;
+#else
+    sh.map_result = VirtualAlloc(NULL, sh.map_size, MEM_COMMIT | MEM_RESERVE, PAGE_READWRITE);
+
+    if (sh.map_result == NULL)
+            goto err;
+
+    memset(sh.map_result, 0, sh.map_size);
```

Comment:
According to this page:

https://docs.microsoft.com/en-us/windows/win32/api/memoryapi/nf-memoryapi-virtualalloc

"Memory allocated by this function is automatically initialized to zero."

So this memset seems redundant?

Reply:
You're right, I'll remove this.